src/main/java/it/gov/pagopa/pu/registry/mapper/pagopa/RegistryMappingService.java

```diff
@@ -0,0 +1,35 @@
+package it.gov.pagopa.pu.registry.mapper.pagopa;
+
+import it.gov.pagopa.pu.registry.utils.RegistryUtils;
+import org.apache.commons.lang3.function.TriFunction;
+import org.springframework.data.util.StreamUtils;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+@Service
+public class RegistryMappingService {
+
+  public <T, R> List<R> mapRegistries(
```

Comment:
this method could be renamed in order to be the main method exposed by the extesions, all the logic is here!
```suggestion
  public List<R> map(T dto)
```